src/plugins/intel_cpu/src/cpu_types.cpp

```diff
@@ -123,6 +123,7 @@ static const TypeToNameMap& get_type_to_name_tbl() {
             { "ScatterElementsUpdate", Type::ScatterElementsUpdate},
             { "ScatterNDUpdate", Type::ScatterNDUpdate},
             { "Interpolate", Type::Interpolate},
+            { "RandomUniform", Type::RandomUniform},
```

Comment:
`RandomUniform` is needed in this PR?

Reply:
This PR based on the RandomUniform PR. Please look at the last commit for now.